src/data-visualisation/decluttering/decluttering.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: collection
+title: "Decluttering and pulling focus"
+description: "Guidance on the importance of decluttering and pulling focus for data visualisation"
+status: DRAFT
+tags: data
+order: 1
```

Comment:
Order here plays a role. This should be different from the order in: `../accessibility/index.md` unless the order doesn't matter here then it should be removed. In theory it should be higher than the 0 in accessibility if you want the data vis articles to have a particular order.

Reply:
Okay will amend. I will need to change the order on accessibility to be higher then as it is not the first page.